Ginger/GingerCore/Resources/PayLoad.js

```diff
@@ -348,6 +348,24 @@ GingerPayLoad.prototype.GetListPayLoad = function()
     }
 }
 
+GingerPayLoad.prototype.GetKeyValue = function () {
+
+    var keyValueObject = {};
+    var byteValue = this.ReadValueType();
+
+    if (byteValue == 8) {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5877667)